client/my-sites/site-monitoring/controller.tsx

```diff
@@ -0,0 +1,38 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { PageViewTracker } from 'calypso/lib/analytics/page-view-tracker';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { SiteMetrics } from './main';
+
+export const siteMetrics: PageJS.Callback = ( context, next ) => {
+	context.primary = (
+		<>
+			<PageViewTracker path="/site-monitoring/:site" title="Site Monitoring" delay={ 500 } />
+			<SiteMetrics />
+		</>
+	);
```

Comment:
As Suggested by @danielbachhuber , we could call it `SiteMonitoring` so it matches the name of this folder `client/my-sites/site-monitoring`

```suggestion
export const siteMonitoring: PageJS.Callback = ( context, next ) => {
	context.primary = (
		<>
			<PageViewTracker path="/site-monitoring/:site" title="Site Monitoring" delay={ 500 } />
			<SiteMonitoring />
		</>
	);
```

Reply:
I am planning to rename the page itself a bit later on when we move over the site logs. For context: site monitoring will be the name for the whole page and it will have sections for site logs and for site metrics. Since for now, I was mostly working only on metrics, I decided to keep it as is and will handle the renaming of the component a bit later on when I am moving site logs in.